controllers/crypto/productsController.js

```diff
@@ -0,0 +1,77 @@
+const logger = require('../../utils/logger')
+const productQuery = require('../../models/crypto/products')
```

Comment:
Simply as `products` will work, or `cryptoProducts` whichever is the better name. Its a reference to the model, not to the query.

It will be changed in the other places in code later.